zenith_utils/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2018"
 
 [dependencies]
 thiserror = "1"
+serde = "1"
```

Comment:
I think this caused the CircleCI build to fail (we need the `derive` feature) although I had no issues locally.
@ericseppanen can I add the workspace_hack crate as a dependency here to solve this, or did you intentionally leave it off `zenith_utils`?

Reply:
That's interesting. The workspace_hack crate was not meant to automatically satisfy all dependencies' features-- we should still make sure that each crate requests the features it will actually use.

I'm disappointed that workspace_hack has solved one issue but added another: a missing feature in a workspace crate Cargo.toml will be un-noticed until you try to build just that crate.

For the moment I would suggest just adding the serde feature here, and we should consider this failure mode a separate bug.